src/main/resources/templates/book.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css"
+          integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+
+    <title>Book Manager</title>
+</head>
+<body>
+<div align="center">
+    <h1>Book List</h1>
+    <br/><br/>
+    <table border="1" cellpadding="10" class="table">
+        <thead>
+        <tr>
+            <th scope="col">Book ID</th>
+            <th scope="col">Title</th>
+            <th scope="col">Author</th>
+            <th scope="col">Customer</th>
+            <th scope="col">Actions</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr scope="row" th:each="book : ${books}">
+            <td th:text="${book.id}">Customer ID</td>
+            <td th:text="${book.title}">Full Name</td>
+            <td th:text="${book.author}">Email Address</td>
+            <td th:text="${book.customer}">Age</td>
+            <td><a th:href="@{'/book/edit/' + ${book.id}}">Edit</a> <a
+                    th:href="@{'/book/delete/' + ${book.id}}">Delete</a></td>
+        </tr>
+        </tbody>
+    </table>
+    <p><a th:href="@{/book/new}">Add New Book</a></p>
+    <p><a th:href="@{/}">View Customers</a></p>
+</div>
+</body>
+</html>
```

Comment:
New line here and in all subsequent :)

Reply:
Hmm, I guess I don't understand what that means.